benchexec/container.py

```diff
@@ -722,6 +740,37 @@ def escape(s):
     )
 
 
+def make_fuse_overlay_mount(exe, mount, lower, upper, work):
+    logging.debug(
+        "Creating overlay mount: target=%s, lower=%s, upper=%s, work=%s",
+        mount,
+        lower,
+        upper,
+        work,
+    )
+
+    def escape(s):
+        return s.replace(b"\\", rb"\\").replace(b":", rb"\:").replace(b",", rb"\,")
```

Comment:
Does fuse-overlayfs have the same restrictions for escaping as kernel overlayfs? Then we should refactor the code to avoid the redundancy.

Reply:
I think fuse-overlayfs has the same restrcitons. According to the fuse-overlayfs [doc](https://github.com/containers/fuse-overlayfs?tab=readme-ov-file#usage), its command format is the same as that of the kernel overlayfs.

Reply:
Can you then please refactor this such that we have only one implementation of it?